phpmyfaq/faq.php

```diff
@@ -336,6 +352,10 @@
             $captchaHelper->renderCaptcha($captcha, 'writecomment', Translation::get('msgCaptcha'), $user->isLoggedIn()),
         'renderComments' => $comment->getComments($faqId, CommentType::FAQ),
         'msg_about_faq' => Translation::get('msg_about_faq'),
+        'bookmarkIcon' => ($bookmark->isFaqBookmark($faqId)===true) ? 'fa fa-bookmark' : 'fa fa-bookmark-o',
+        'bookmarkLink' => ($bookmark->isFaqBookmark($faqId)===true) ? sprintf('index.php?action=faq&bookmark_action=remove&id=%s', $faqId) : sprintf('index.php?action=faq&bookmark_action=add&id=%s', $faqId),
+        'msgAddBookmark' => ($bookmark->isFaqBookmark($faqId)===true) ? Translation::get('removeBookmark') : Translation::get('msgAddBookmark'),
+        'alert' => (isset($alertMessage)) ? $alertMessage : '',
```

Comment:
I would rename it to bookmarkAlert to make it more clear

Reply:
Just the variable alertMessage to bookmarkAlert or the {{ variable }} as well?